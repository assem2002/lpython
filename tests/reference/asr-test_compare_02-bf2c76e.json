{
    "basename": "asr-test_compare_02-bf2c76e",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_compare_02.py",
    "infile_hash": "16c5516272fbd5a54c9039bfe614e3813425f92826025c0ed5671341",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_compare_02-bf2c76e.stderr",
    "stderr_hash": "f37d7d40e45b8e7fd25d963085ff1c0060fc744da76b168ea84d341f",
    "returncode": 2
}