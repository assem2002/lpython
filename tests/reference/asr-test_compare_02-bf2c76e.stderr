semantic error: Type mismatch in comparison operator, the types must be compatible
 --> tests/errors/test_compare_02.py:6:11
  |
6 |     print(x > y)
  |           ^   ^ type mismatch ('i32' and 'u32')


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
